lib/db_helper.rb

```diff
@@ -161,4 +163,78 @@ def self.truncate(sql, table, column)
       sql
     end
   end
+
+  def self.build_remap_query_parts(table, columns, skip_max_length_violations)
+    columns.each_with_object(
+      { updates: [], conditions: [], skipped_sums: [], length_constrained_columns: [] },
+    ) do |column, parts|
+      replace = "REPLACE(\"#{column[:name]}\", :from, :to)"
```

Comment:
I see you avoided escaping the quote a couple of lines below. It might be a good idea here as well.

```suggestion
      replace = %|REPLACE("#{column[:name]}", :from, :to)|
```